templates/windows/chocolateyInstall.ps1.erb

```diff
@@ -0,0 +1,5 @@
+<%- dirnames = get_directories.map {|d| d.path } -%>
+<%- dirnames.each do |dir| -%>
+  <%- windows_path = platform.convert_to_windows_path(dir) -%>
+Copy-Item <%= "#{windows_path.sub("C:", "C:/ProgramData/chocolatey/lib/#{project.name}")}" %> <%= windows_path %> -recurse
```

Comment:
`"C:/ProgramData/chocolatey/lib/#{project.name}"` is considered bad form as this may not be where a package is installed (if the chocolatey install directory is different).

Instead you should prefer the path to the folder where the chocolateyInstall.ps1 is - `$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"`